src/view/com/composer/select-language/SelectPostLanguagesDialog.tsx

```diff
@@ -0,0 +1,418 @@
+                size={16}
+                color={
+                  search.length > 0
+                    ? t.palette.primary_500
+                    : t.atoms.text_contrast_low.color
+                }
+                style={{
+                  marginLeft: 'auto',
+                  marginRight: 'auto',
+                }}
+              />
+            </View>
+          </TextField.Root>
+        </View>
+        {showSearchCancel && (
+          <Button
+            label="Cancel"
+            size="small"
+            variant="ghost"
+            color="secondary"
+            style={[a.px_sm]}
+            onPress={() => setSearch('')}>
+            <ButtonText>
+              <Trans>Cancel</Trans>
+            </ButtonText>
+          </Button>
+        )}
+      </View>
+    </View>
+  )
+
+  const isCheckedRecentEmpty =
+    displayedLanguages.checkedRecent.length > 0 ||
+    displayedLanguages.uncheckedRecent.length > 0
+
+  const isDisplayedLanguagesEmpty = displayedLanguages.all.length === 0
+
+  const flatListData = [
+    ...(isCheckedRecentEmpty ? [{type: 'header', label: 'Recently used'}] : []),
+    ...displayedLanguages.checkedRecent.map(lang => ({type: 'item', lang})),
+    ...displayedLanguages.uncheckedRecent.map(lang => ({type: 'item', lang})),
+    ...(isDisplayedLanguagesEmpty
+      ? []
+      : [{type: 'header', label: 'All languages'}]),
+    ...displayedLanguages.all.map(lang => ({type: 'item', lang})),
+  ]
+
+  const {gtMobile} = useBreakpoints()
+
+  return (
+    <>
+      <Dialog.Close />
+
+      <ErrorBoundary
+        renderError={error => <DialogError details={String(error)} />}>
+        <View
+          style={[
+            web({
+              width: gtMobile ? WEB_DIALOG_WIDTH : MOBILE_DIALOG_WIDTH,
+              maxWidth: gtMobile ? WEB_DIALOG_WIDTH : MOBILE_DIALOG_WIDTH,
+              position: a.relative,
+            }),
+            web(a.flex_1),
+            isNative && {
+              flexGrow: 1,
+              minHeight: '100%',
+            },
+          ]}>
+          <Toggle.Group
+            values={checkedLanguagesCode2}
+            onChange={setCheckedLanguagesCode2}
+            type="checkbox"
+            maxSelections={3}
+            label="languageSelection">
+            <Dialog.InnerFlatList
+              ref={listRef}
+              data={flatListData}
+              renderItem={({item, index}) => {
+                if (item.type === 'header') {
+                  const isAllLanguages = item.label === 'All languages'
+
+                  return (
+                    <Text
+                      style={[
+                        a.px_0,
+                        a.py_md,
+                        a.font_bold,
+                        a.text_xs,
+                        t.atoms.text_contrast_low,
+                        a.pt_2xl,
+                        isAllLanguages ? a.pt_2xl : a.pt_xl,
+                      ]}>
+                      <Trans>{item.label}</Trans>
```

Comment:
```suggestion
                      {item.label}
```
The `Trans` tags aren't needed here as there would be no translatable text :)